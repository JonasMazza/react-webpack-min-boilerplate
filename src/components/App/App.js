import React, { Component } from 'react';

import catImg from './assets/cat.png';
import './styles/App.css';

class App extends Component {
  render() {
    return (
      <div className='greeting'>
        <img alt='' src={catImg} />
        <h1>There you go!!</h1>
      </div>
    );
  }
}

export default App;