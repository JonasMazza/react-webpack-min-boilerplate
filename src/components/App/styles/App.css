@font-face {
  font-family: 'shinkregular';
  src: url('../assets/fonts/shink-webfont.woff2') format('woff2'),
       url('../assets/fonts/shink-webfont.woff') format('woff');
  font-weight: normal;
  font-style: normal;
}

.greeting {
  display: flex;
  flex-direction: column;
}

.greeting h1 {
  color: red;
  font-size: 64px;
  text-align: center;
  font-family: 'shinkregular';
}

.greeting img {
  margin: 10px auto;
}