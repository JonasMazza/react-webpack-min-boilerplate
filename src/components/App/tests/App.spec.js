import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';

describe('App', () => {
  test('should match snapshot', () => {
    const app = renderer.create(<App />);
    expect(app).toMatchSnapshot();
  });
});