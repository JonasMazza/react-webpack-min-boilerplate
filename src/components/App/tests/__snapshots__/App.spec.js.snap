// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['App should match snapshot 1'] = `
<div
  className="greeting"
>
  <img
    alt=""
    src="test-file-stub"
  />
  <h1>
    There you go!!
  </h1>
</div>
`;
